### DISCUSSION TITLE
Finishing the Markdown conversion

### DISCUSSION THREAD

**Discussion by wbamberg** [mdn/mdn-community#576] [Content]
After https://github.com/mdn/content/issues/8741, this spreadsheet: https://docs.google.com/spreadsheets/d/1gIW85rRxPkgsGjYw_5wLa_kGgMeOgnfYrpWytOMD8Lw/edit#gid=0 lists the remaining HTML files. I'm going to propose we convert the rest in the following sections, in the following order:

Area | Page count
-- | --
web/html (done) | 234
web/svg (done) | 376
learn | 362
web/accessibility | 104
glossary | 538
mozilla/add-ons | 634
(the rest) | 586

I'll file a tracking bug to prep and convert each of these areas.

Note that this _excludes_ mozilla/projects/nss and devtools, which we don't maintain and which we plan to remove from MDN, as I understand it: (NSS reference: https://github.com/mdn/content/issues/2669, devtools reference, a less definitive: https://github.com/mdn/content/pull/8215#issuecomment-904038157).

I believe other people have expressed an interest in (and actually started the work of) converting some of these areas. If so I'd be happy for you to claim these areas in responses to this post.

******

**Update:** removed devtools from the list.

**Comment by hamishwillee** [comment]
@wbamberg That's cool. I've been looking at Learn/Server-Side and Glossary as a background task when my brain is gone. While I'm going to continue on those, they're a fallback, not a main task. So anyone else who wants to work on them before I get them done is quite welcome to do so.

**Comment by wbamberg** [comment]
Tracking bug for HTML: https://github.com/mdn/content/issues/8961

**Comment by wbamberg** [comment]
Tracking bug for SVG: https://github.com/mdn/content/issues/8969

**Comment by wbamberg** [comment]
@ddbeck , @caitmuenster , @rebloor : as you might know we're currently converting MDN docs from HTML files into Markdown. One of the big areas left is the add-ons docs: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons .

I'm happy to do this conversion if you like (will ask for review from Richard and anyone else you like) but since you are the primary maintainers of these docs I think you should get to say whether you want them in Markdown, or would prefer that they stay in HTML. I'm happy to talk about the implications of making this change, either in a chat or in this discussion. I'm not sure how much it helps, but there is a page explaining what the new format is like: https://developer.mozilla.org/en-US/docs/MDN/Contribute/Markdown_in_MDN .

FWIW I think conversion to Markdown would be a good thing for the Add-ons docs, and as far as I can remember there shouldn't be any big problems with the conversion.

**Comment by wbamberg** [comment]
@ddbeck , similarly to https://github.com/mdn/content/discussions/8960#discussioncomment-1410619: do the maintainers of the devtools documentation (https://developer.mozilla.org/en-US/docs/Tools) want it in Markdown? The same applies: I'm happy to do the conversion but don't feel like I should make the call about whether it is desirable.

**Reply by wbamberg** [reply]
Consensus is *not* to convert the Tools docs.

**Comment by SphinxKnight** [comment]
Tracking issue for Learning Area: https://github.com/mdn/content/issues/9218

**Comment by SphinxKnight** [comment]
Tracking issue for Glossary: https://github.com/mdn/content/pull/9361

**Comment by SphinxKnight** [comment]
Issue for web/accessibility (I think?) https://github.com/mdn/content/pull/9631

**Comment by SphinxKnight** [comment]
Tracking issue for WebAssembly: https://github.com/mdn/content/pull/9721

**Comment by SphinxKnight** [comment]
Tracking issue for Mozilla/Add-ons: https://github.com/mdn/content/issues/9842

**Comment by wbamberg** [comment]
Tracking issue for Mozilla/Firefox prep: https://github.com/mdn/content/issues/9925

**Comment by SphinxKnight** [comment]
PR for Games conversion: https://github.com/mdn/content/pull/9659

**Comment by pepelsbey** [comment]
Closing as outdated/irrelevant.
